src/main/java/test/OlegLoginTest.java

```diff
@@ -0,0 +1,21 @@
+package main.java.test;
+
+import static org.testng.Assert.*;
+import main.java.pages.header.UnlogUserInfo;
+import main.java.pages.login.LoginForm;
+import main.java.tools.WebDriver;
+import org.openqa.selenium.By;
+import org.testng.annotations.Test;
+
+public class OlegLoginTest {
+	@Test
+	public void testLoginForm(){
+		WebDriver webDriver = new WebDriver ();
+		UnlogUserInfo unlogUserInfo = new UnlogUserInfo ();
+		unlogUserInfo.clickSignIn ();
+		LoginForm loginForm = new LoginForm (  );
```

Comment:
why are we creating one webdriver in test method and then create another in initLogin?

Reply:
it was just for test, i will correct it right now